### PR TITLE
refactor: update prepare-test-net script and include sudo / oracle in…

### PR DESCRIPTION
**Pull request by gregdhill**
… endowed_accounts

Signed-off-by: Gregory Hill <gregorydhill@outlook.com>

### CODE REVIEW

File: parachain/src/chain_spec.rs [hunk 1]
```diff
@@ -275,6 +275,10 @@ pub fn staging_testnet_config(id: ParaId) -> TestnetChainSpec {
                     ]),
                 ],
                 vec![
+                    // 5Ec37KSdjSbGKoQN4evLXrZskjc7jxXYrowPHEtH2MzRC7mv (//sudo/1)
+                    get_account_id_from_string("5Ec37KSdjSbGKoQN4evLXrZskjc7jxXYrowPHEtH2MzRC7mv"),
+                    // 5ECj4iBBi3h8kYzhqLFmzVLafC64UpsXvK7H4ZZyXoVQJdJq (//oracle/1)
+                    get_account_id_from_string("5ECj4iBBi3h8kYzhqLFmzVLafC64UpsXvK7H4ZZyXoVQJdJq"),
```
Comment (sander2) [inline parachain/src/chain_spec.rs:281]: Should we remove the account/1-5 and add the vault?

Comment (gregdhill) [reply parachain/src/chain_spec.rs:281]: These are just random accounts we can use for the vaults

**Comment by gregdhill** [comment]
> Is it possible to do `author_insertKey` somewhere in the code so we don't have to manually do it each time we reset?

No because we'd then somehow need to expose the private (root) key in the chain spec. @ns212 already has automation for this provided we use the right keys.

File: parachain/src/chain_spec.rs (hunk 1 above)
Comment (sander2) [reply parachain/src/chain_spec.rs:281]: Not sure what is most convenient, I'll leave it up to you

**Review by sander2 — APPROVED** [review]

**Pull request merged**